foxglove/console/api.go

```diff
@@ -34,18 +34,19 @@ type UploadResponse struct {
 type StreamRequest struct {
 	ImportID     string     `json:"importId,omitempty"`
 	DeviceID     string     `json:"deviceId,omitempty"`
```

Comment:
While we're here, maybe:

```suggestion
	DeviceID     string     `json:"device.id,omitempty"`
```